packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -216,6 +221,21 @@ const BlockComponent = forwardRef(
 				{ children }
 			</Animated>
 		);
+
+		if ( supportAlignments ) {
+			return (
+				<div
+					className={ classnames( 'wp-block wp-align-wrapper', {
+						[ `wp-align-${ attributes.align }` ]: !! attributes.align,
+					} ) }
+					data-align={ attributes.align }
```

Comment:
Why are we adding the data attributes here if we're going with classes? I think this will only make things more confusing as it's the old way of styling alignment in the editor.